Library/Formula/camlp5.rb

```diff
@@ -13,20 +13,41 @@ class Camlp5 < Formula
 
   deprecated_option "strict" => "with-strict"
   option "with-strict", "Compile in strict mode"
+  option "with-doc", "Install the html and info documentation"
+  option "with-tex", "Install the pdf, ps, and tex documentation"
 
   depends_on "ocaml"
+  depends_on "gnu-sed" => :build if build.with?("doc") || build.with?("tex")
+  depends_on :tex => [:build, :optional]
 
   def install
-    if build.with? "strict"
-      strictness = "-strict"
-    else
-      strictness = "-transitional"
+    args = ["--prefix", prefix, "--mandir", man]
+    args << "--transitional" if build.without? "strict"
+
+    if build.with?("tex") && (ps2pdf = which("ps2pdf")).nil?
+      saved_path = ENV["PATH"]
+      ENV.append_path "PATH", "/usr/local/bin" # known install location by MacTeX
+      odie "Couldn't find ps2pdf" if (ps2pdf = which("ps2pdf")).nil?
+      ENV["PATH"] = saved_path
```

Comment:
I do see that; More than anything I want to avoid the precedent of allowing workarounds like this. There's a significant amount of mission creep potential in just needing to re-add the most common `HOMEBREW_PREFIX/bin` to the `$PATH` _temporarily_ :cry:.

Is there a TeX installation that doesn't supply `ps2pdf`?